kyo-bench/src/main/scala/kyo/bench/ChannelBench.scala

```diff
@@ -26,7 +26,7 @@ class ChannelBench extends BaseBench:
                 _       <- Async.run(channel.closeAwaitEmpty)
                 count   <- channel.streamUntilClosed(maxChunkSize).into(Sink.count)
             yield count
-        val count = IO.Unsafe.evalOrThrow(Async.run(x).flatMap(_.block(Duration.Infinity))).getOrThrow
+        val count = Sync.Unsafe.evalOrThrow(Async.run(x).flatMap(_.block(Duration.Infinity))).getOrThrow
```

Comment:
did we miss the rename in the bench module? it's odd since the build isn't failing

Reply:
indeed, I have since that in logs.